src/pages/K8s.vue

```diff
@@ -435,6 +446,30 @@ export default {
     handleError(key, message) {
       this.handleNotification('error', key, message);
     },
+    confirm(message) {
+      return ipcRenderer.invoke(
+        'show-message-box',
+        {
+          message,
+          type:     'question',
+          title:    'Rancher Desktop - Kubernetes Settings',
+          buttons:  ['Ok', 'Cancel'],
```

Comment:
Text should be `OK` (see src/assets/translations/en-us.yaml)

Reply:
I just opted to use i18n so we don't have to guess.